main.py

```diff
@@ -101,11 +84,64 @@ def process_data():
     print(df4.to_string())
     df4.to_csv("filtered_data.csv")
 
+def cal_dividend_increament(div_obj, p):
+    # This function calculate the average exponential increments of the dividend
+    # and check wheter it is monotonically increasing and if it persistent with maximal time between dividends of 100 days
```

Comment:
great documentation
the pythonic was is to use docstring
that is
"""
your documentation...
"""